src/types.typ

```diff
@@ -11,9 +11,12 @@
 #let color = base-type.with(name: "color", types: (color,))
 #let content = base-type.with(name: "content", types: (content, str))
 #let date = base-type.with(name: "date", types: (datetime,))
+#let gradient = base-type.with(name: "gradient", types: (gradient,))
 #let number = base-type.with(name: "number", types: (float, int))
 #let integer = number.with(name: "integer", types: (int,))
 #let floating-point = number.with(name: "float", types: (float,))
+#let stroke = base-type.with(name: "stroke", types: (stroke,))
+#let version = base-type.with(name: "version", types: (version,))
```

Comment:
I think we could still add a few other basic types here. [Here](https://github.com/typst-community/mantodea/blob/14d2ee86ae7b439c37c511d14e1221fdec76a530/src/theme.typ#L29-L73)'s a list of all built-in types.

Reply:
It turns out I missed the `symbol` type.

Reply:
Updated original post to track all these